admin/javascript/src/sections/campaign-admin/item.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+/**
+ * Describes an individual campaign item
+ */
+class CampaignItem extends SilverStripeComponent {
+  render() {
+    let thumbnail = '';
+    let badge = '';
+    const item = this.props.item;
+
+    // change badge
+    switch (item.ChangeType) {
+      case 'created':
+        badge = <span className="label label-warning">Draft</span>;
+        break;
+      case 'modified':
+        badge = <span className="label label-warning">Modified</span>;
+        break;
+      case 'deleted':
+        badge = <span className="label label-error">Removed</span>;
+        break;
+      case 'none':
+      default:
+        badge = <span className="label label-success item_visible-hovered">Already published</span>;
+        break;
+    }
+
+    // Linked items
+    let links = <span className="list-group-item__linked item_visible-hovered">[lk] 3 links</span>;
```

Comment:
@clarepaul is just implementing the "link" designs as shown on https://projects.invisionapp.com/boards/9GX68PTXUQFH#/1547737/71700348, shouldn't block this PR